Add Visualization Processing Docs (#1769)
hubmapconsortium/portal-ui: visualization-docs/README.md

<details><summary>Has the data been validated via ingest-validation-tools and confirmed to be viewable using Avivator (which loads data almost identically to what is in the portal)?</summary>

> To this end, we should ask the TMC to follow the instructions below for viewing their data in Avivator to make sure it looks right (should only need to be done for a single representative file): https://github.com/hms-dbmi/viv/tree/master/tutorial

Reviewer: ```suggestion
If so, we should ask the TMC to follow the instructions below for viewing their data in Avivator to make sure it looks right (should only need to be done for a single representative file): https://github.com/hms-dbmi/viv/tree/master/tutorial
```

In the above instructions they should only need to a) run the bioformats2raw-raw2ometiff pipeline and then b) drag-and-drop or select the input file using the "CHOOSE A FILE" button on avivator.gehlenborglab.org. There is no need for a web server.

If there is a z or t stack to the data, ensure that each "stack" is uploaded as a single file.

If it is valid in these three senses (viewable in Avivator locally, passes `ingest-validation-tools`, and "stacks" are uploaded as single files), then ingestion may be done and pipeline processing may proceed.

</details>

> <details><summary>Is there spot data?</summary>

Reviewer: Will "spot data" be clear to readers? Do you mean "point data", ie, properties are associated a particular coordinate?

Author: I think "spot" data is a term used for FISH assay outputs (that I have heard).  I'll add some clarification like "(for example is this a FISH assay with resolved locations of probes that you want to show)"

> If the answer is "yes," we should run the image pyramid pipeline + offsets on the appropriate imaging data.  We currently do not have a pipeline for visualizing spot data.  Create a new class that inherits from ViewConf to visualize the data (raw imaging + spot data) when such a pipeline is created.  If there is segmentation data coming from the TMC or elsewhere, then that will need to be both processed (via <a href="https://github.com/hubmapconsortium/portal-containers/tree/master/containers/sprm-to-anndata">sprm-to-anndata.cwl from portal-containers</a> or a different pipeline that ideally outputs zarr-backed AnnData) and visualized as well

Reviewer: ```suggestion
If the answer is "yes," we should run the image pyramid pipeline + offsets on the appropriate imaging data.  We currently do not have a pipeline for visualizing spot data.  Create a new class that inherits from ViewConf to visualize the data (raw imaging + spot data) when such a pipeline is created.  If there is segmentation data coming from the TMC or elsewhere, then that will need to be both processed (via <a href="https://github.com/hubmapconsortium/portal-containers/tree/master/containers/sprm-to-anndata">sprm-to-anndata.cwl from portal-containers</a> or a different pipeline that ideally outputs zarr-backed AnnData) and visualized as well.
```
